config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+
+  get 'registrations/sign_up_params'
+  mount_devise_token_auth_for 'User', at: 'api/v1'
```

Comment:
You can do something like this, and avoid explicit routes.
```suggestion
  mount_devise_token_auth_for 'User', at: 'api/v1', controllers: {
    registrations: 'api/v1/registrations',
```

https://guides.rubyonrails.org/routing.html#specifying-a-controller-to-use